### ISSUE TITLE
can't get linter working

### ISSUE THREAD

**Issue by jmls** [SublimeLinter/SublimeLinter-gjslint#5]
using sublime text3 

I think that I've followed all the instructions, both here and at sublimelinter, debug says the following

```html
reloading plugin SublimeLinter-gjslint.linter
SublimeLinter: gjslint linter loaded 
reloading plugin SublimeLinter.commands
reloading plugin SublimeLinter.sublimelinter plugins loaded
SublimeLinter: debug mode: on 
```

which tells me

```html
which gjslint
/usr/local/bin/gjslint
```

sublimelinter says that gjslinter is enabled

*but* I cannot lint any file - the option to "lint this view" is greyed out, and any lint mode (load/save , background etc) does not show any errors (I know that my code is not _that_ good ;) )

what am I missing

**Comment by xakutin** [comment]
Can you put your SublimeLinter User settings? 
I suppose the plugin it's not disabled:
```
...
"gjslint": {
  "@disable": false,
...
```

In your debug lines I miss this:
```
SublimeLinter: gjslint activated: /usr/local/bin/gjslint 
```
When you open a js file.

**Comment by jmls** [comment]
sorry, I must have missed that during copy & paste ...

```html
eloading Packages/User/Package Control.sublime-settings
reloading Packages/User/Preferences.sublime-settings
Package Control: Upgraded SublimeLinter to v3.4.38
ignored packages updated to: ["Vintage"]
reloading Packages/SublimeLinter/SublimeLinter.sublime-settings
reloading plugin SublimeLinter.commands
reloading plugin SublimeLinter.sublimelinter
SublimeLinter: debug mode: off 
SublimeLinter: daemon detected a reload 
generating syntax summary
generating syntax summary
reloading Packages/User/Package Control.sublime-settings
reloading Packages/User/Preferences.sublime-settings
reloading Packages/User/Preferences.sublime-settings
reloading Packages/User/SublimeLinter.sublime-settings
SublimeLinter: gjslint activated: /usr/local/bin/gjslint 
```

so, it looks like it's loaded, it says "generating syntax summary" but I can't see any errors

**Comment by jmls** [comment]
in my user settings, I have this

```json
"lint_mode": "background",
        "linters": {
            "gjslint": {
                "@disable": false,
                "args": [],
                "disable": "",
                "excludes": [],
                "jslint_error": "",
                "max_line_length": null
            },
            "jshint": {
                "@disable": true,
                "args": [],
                "excludes": []
            }
        },
```

**Comment by xakutin** [comment]
Can you paste all your user settings?
Are you using a .gjslintrc file?
Have you try uninstalling sublimelinter-gjslint, sublimelinter-jshint, sublimelinter, and reinstalling sublimelinter and sublimelinter-gjslint?

With the debug mode you should see debug lines like this:
```
SublimeLinter: gjslint: test.js ['/usr/local/bin/gjslint', '--nobeep', '--nosummary', '--flagfile', '/home/xakutin/.gjslintrc'] 
SublimeLinter: gjslint output:
----- FILE  :  /tmp/SublimeLinter3-xakutin/test.js -----
Line 1, E:0002: Missing space before "="
Found 1 errors, including 0 new errors, in 1 files (0 files OK).

Some of the errors reported by GJsLint may be auto-fixable using the script
fixjsstyle. Please double check any changes it makes and report any bugs. The
script can be run by executing:

fixjsstyle --nosummary --flagfile /home/xakutin/.gjslintrc /tmp/SublimeLinter3-xakutin/test.js 
```
Please, activate the debug mode and also the jshint linter.

**Comment by jmls** [comment]
yeah, ok, so something really weird is happening .. I just edited an html file, and got

```html
SublimeLinter: gjslint activated: /usr/local/bin/gjslint 
reloading Packages/User/SublimeLinter.sublime-settings
reloading Packages/User/SublimeLinter.sublime-settings
SublimeLinter: gjslint: codeEditor.html ['/usr/local/bin/gjslint', '--nobeep', '--nosummary', '--flagfile', '/opt/development/nodegloo/.gjslintrc'] 
SublimeLinter: gjslint output:
----- FILE  :  /tmp/SublimeLinter3-jmls/codeEditor.js -----
Line 1, E:0001: Extra space at end of line
Line 1, E:0300: File does not end with new line.  ( $('.collapse-card').paperCollapse(); )
Found 2 errors, including 0 new errors, in 1 file (0 files OK).

Some of the errors reported by GJsLint may be auto-fixable using the script
fixjsstyle. Please double check any changes it makes and report any bugs. The
script can be run by executing:

fixjsstyle --nosummary --flagfile /opt/development/nodegloo/.gjslintrc /tmp/SublimeLinter3-jmls/codeEditor.js 
```
odd thing #1 : codeEditor is codeEditor.html, not codeEditor.js 
odd thing #2 : I can't get the lint to run on any other file, html or js ;(

my .gjslintrc is

```
--exclude_directories=lib/client/public,node_modules,temp
--exclude_files=Gruntfile.js
--max_line_length=120
--disable=0110
```

and my settings are

```
{
    "user": {
        "@disable": false,
        "debug": true,
        "delay": 0.25,
        "error_color": "D02000",
        "gutter_theme": "Packages/SublimeLinter/gutter-themes/Default/Default.gutter-theme",
        "gutter_theme_excludes": [],
        "lint_mode": "background",
        "linters": {
            "gjslint": {
                "@disable": false,
                "args": [],
                "disable": "",
                "excludes": [],
                "jslint_error": "",
                "max_line_length": null
            },
            "jshint": {
                "@disable": true,
                "args": [],
                "excludes": []
            }
        },
        "mark_style": "fill",
        "no_column_highlights_line": false,
        "passive_warnings": false,
        "paths": {
            "linux": [],
            "osx": [],
            "windows": []
        },
        "python_paths": {
            "linux": [],
            "osx": [],
            "windows": []
        },
        "rc_search_limit": 3,
        "shell_timeout": 10,
        "show_errors_on_save": true,
        "show_marks_in_minimap": true,
        "syntax_map": {
            "html (django)": "html",
            "html (rails)": "html",
            "html 5": "html",
            "php": "html",
            "python django": "python"
        },
        "warning_color": "DDB700",
        "wrap_find": true
    }
}
```

i will uninstall and reinstall as suggested

thanks

**Comment by jmls** [comment]
uninstall / reinstall gives same problems.

the same html file does lint though ;)

**Comment by xakutin** [comment]
I've tested with your config (user config and .gjslintrc) and it lints my files.
Can you try this:
- make a folder ~/testing/
- add a file ~/testing/file1.js
- in this file: var a=1;
Check if it lints the error.

Apart from that, which are the versions of the sublimelinter and sublimelinter-gjslint?
In my case:
SublimeLinter v.3.4.38
SublimeLinter-gjslint v.1.0.12

**Comment by skj3gg** [comment]
What is the stated syntax of the js files that you try to lint, that don't work.
